config/rewrites.d/support.rackspace.com.json

```diff
@@ -17,9 +17,9 @@
           "status": 302
       },
       {
-          "description": "white-paper index to all articles",
+          "description": "redirect old white-paper links to new whitepapers landing page",
           "from": "^\\/white-paper\\/$",
```

Comment:
Did you want to redirect the homepage, or were you looking to redirect any and all old white paper links to the new white papers landing page? Currently, this rewrite just sends https://support.rackspace.com/white-paper/ to the new landing page. If you want to redirect all links, you need to replace `$` with `?(.*)$`. (See https://one.rackspace.com/display/devdoc/Redirects and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions for more information.)